sql/core/src/test/resources/sql-tests/results/show-tables.sql.out

```diff
@@ -128,6 +128,7 @@ View Schema Mode: BINDING
 Schema: root
  |-- e: integer (nullable = true)
 
+
```

Comment:
do you which field introduces this unnecessary blank line?

Reply:
it seems to be the schema field [here](https://github.com/apache/spark/blob/7bc8e99cde424c59b98fe915e3fdaaa30beadb76/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala#L647) 

> if (schema.nonEmpty) map += "Schema" -> JString(schema.treeString)